src/pages/AdminBooksPage/BooksTable/BooksTableSkeleton/BooksTableSkeleton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Skeleton, TableBody, TableCell, TableRow } from "@mui/material";
+
+const BooksTableSkeleton = () => {
```

Comment:
This should be in a separate PR, if we merge it like this, problems with skeleton would require revert of delete book feature